README.md

```diff
@@ -1 +1,245 @@
+
+```objc
+// using getter methods (valid)
+
+NSString *welcome = @"Welcome to the Flatiron School!";
+NSString *welcomeUC = [welcome uppercaseString];
+
+NSLog(@"%@", welcomeUC);
+```
+This is equivalent to:
+
+```objc
+// using dot notation (preferred)
+
+NSString *welcome = @"Welcome to the Flatiron School!";
+
+NSLog(@"%@", welcome.uppercaseString);
+```
+These will both print: `WELCOME TO THE FLATIRON SCHOOL!`.
+
+## The `readonly` & `readwrite` Property Options
+
+For information that needs to be publicly visible, but still under the jurisdiction of the owning class to overwrite or change, the `readonly` and `readwrite` property options can be used.
+
+The `readonly` property option does just that: it makes the property unwritable, or "read only". It accomplishes this by creating the property *without* an associated "setter" method.
+
+To create a read-only property, the `readonly` property option must be added to the public declaration of the property. Since there is no setter method generated, in order to write to the property within the class that owns, the property must be *redeclared* in the private `@interface` section as a `readwrite` property that generates a setter with private visibility.
+
+Since the frequency window in the car radio that we're modeling with the `FISRadio` class is readable but only interactive through the "tuning" dial, it makes sense to express the `amFrequency` and `fmFrequency` properties as `readonly`. We can then provide public methods that privately write to the `readonly` properties in a manner that we can control from within the owning class:
+
+```objc
+//  FISRadio.h
+
+#import <Foundation/Foundation.h>
+#import <CoreGraphics/CGBase.h>   // imports CGFloat
+
+@interface FISRadio : NSObject
+
+@property (nonatomic, readonly) CGFloat amFrequency;
+@property (nonatomic, readonly) CGFloat fmFrequency;
+
+{...}
+
+- (void)defaultFrequencies;
+- (void)increaseFrequencies;
+- (void)decreaseFrequencies;
+
+@end
+```
+
+```objc
+//  FISRadio.m
+
+#import "FISRadio.h"
+
+@interface FISRadio ()
+
+@property (nonatomic, readwrite) CGFloat amFrequency;
+@property (nonatomic, readwrite) CGFloat fmFrequency;
+
+{...}
+
+@end
+
+@implementation
+
+- (void)defaultFrequencies {
+    self.amFrequency = 150.0;
+    self.fmFrequency = 91.0;
+}
+
+- (void)increaseFrequencies {
+    if (self.amFrequency + 0.5 <= 160.0) {
+        self.amFrequency += 0.5;
+        self.fmFrequency += 0.1;
+    }
+}
+
+- (void)decreaseFrequencies {
+    if (self.amFrequency - 0.5 >= 53.0) {
+        self.amFrequency -= 0.5;
+        self.fmFrequency -= 0.1;
+    }
+}
+
+@end
+```
+
+**Note:** *The* `readwrite` *keyword in the private redeclaration of  the property isn't strictly necessary since it is the implicitly default, however it is considered a best practice to indicate that the property's public visibility is meant to be* `readonly`.
```

Comment:
This may be one of those cases where it's not even necessary to tell them they have an option.